installer/Migration/V5_4_0/Migration.php

```diff
@@ -192,6 +192,35 @@ public function up(): void
         );
         $this->getSchemaHelper()->addForeignKey('ohrm_enforce_password', $foreignKeyConstraint);
 
+        if (!$this->getSchemaHelper()->tableExists(['ohrm_expense'])) {
```

Comment:
fix this also
https://github.com/orangehrm/orangehrm/blob/5.4/installer/Migration/V5_4_0/Migration.php#L48-L49